lib/yggdrasil/player.ex

```diff
@@ -12,9 +13,16 @@ defmodule Yggdrasil.Player do
     Supervisor.add_player user_id, [owner_pid, push_msg]
   end
 
-  def run_cmd(user_id, cmd = %Message{type: :command}) do
-    player_pid = Registry.get_player user_id
-    GenServer.cast(player_pid, {:run_cmd, cmd})
+  def run_cmd(user_id, cmd) do
+    # horrible terrible temporary implementation
+    # room would eventually do all of this
+    ctxt = %Yggdrasil.Room.Context{ player: user_id }
```

Comment:
So this the part I don't like I think the context should start in the player process and the player process dispatch it to the room. The room can still run the command.

We have a way to resolve user_id to a player process which is good, and I think that is where user_id should stop otherwise it's bleeding to far in.

My thoughts are that if we do this from the player process one, we know the pid of the player so for example `player: self()` can be done, no need for a look up here, and I think the player process would be aware of the room it's in. I sort envision the state of the player process to contain the current room pid, and it talks to the room that way and when the player moves it's just given a new pid to talk to. 

This gets rid of having to have any sort of global look up for rooms etc I think.